src/Plugin/resource/DataSource/DataSourceInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\restful\Plugin\resource\DataSource\DataSourceInterface.
+ */
+
+namespace Drupal\restful\Plugin\resource\DataSource;
+
+interface DataSourceInterface {
+
+  /**
+   * Get the account.
+   *
+   * @return object
+   *   The fully loaded account.
+   */
+  public function getAccount();
+
+  /**
+   * Get the wrapper.
+   *
+   * @return mixed
+   *   The entity metadata wrapper describing the entity. Every data source will
```

Comment:
``The entity metadata wrapper describing the entity`` is just an example of a wrapper, so I think we can remove this line.